server/template.js

```diff
@@ -1,37 +1,84 @@
 import marinate from 'marinate';
 import { ONLINE_COLOR } from '../common/constants';
 
+const preloadImages = [
+  '/assets/images/sttm_logo.png',
+  '/assets/images/logo-192x192.png',
+];
+const preloadImagesHTML = preloadImages
+  .map(i => `<link async rel="preload" href="${i}" as="image">`)
+  .join('\n');
+
+const preloadFonts = [
+  '/assets/fonts/AnmolLipiSG.ttf?v=1',
+  '/assets/fonts/GurbaniAkharHeavyTrue.ttf?v=1',
+];
+const preloadFontsHTML = preloadFonts
+  .map(f => `<link async rel="preload" href="${f}" as="font"></link>`)
```

Comment:
Is there any reason this isn't self-closing?

Reply:
Nice catch, thank you!